case_studies/dc2/train_dc2.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# run the galaxy parameters and psf parameters extration notebook
+jupyter nbconvert --execute DC2_galaxy_psf_params.ipynb --to html
+
```

Comment:
With those two packages added, I'm getting this error
```
ValueError: `base_dir` ^/lsstdesc-public/dc2/run2.2i-dr6-v4/truth_match is not a valid directory
```

from the call to `jupyter nbconvert --execute DC2_galaxy_psf_params.ipynb --to html`

Reply:
It's from this line:
```
 truth_cat = GCRCatalogs.load_catalog('desc_dc2_run2.2i_dr6_truth') 
```

Maybe we need to specify the path to the DC2 data?